diff --git a/dist/vue-repl.js b/dist/vue-repl.js
index fe7b7b11bda270071162000a34d9e41c7585e016..7a1f5e28fa1df5f1e33d6aba8af563c309b7810f 100644
--- a/dist/vue-repl.js
+++ b/dist/vue-repl.js
@@ -1,4 +1,4 @@
-import './vue-repl.css'
+// import './vue-repl.css'
 import { defineComponent, computed, useTemplateRef, inject, reactive, createElementBlock, openBlock, normalizeClass, unref, createElementVNode, normalizeStyle, renderSlot, withModifiers, withDirectives, toDisplayString, vShow, ref, watch, createBlock, Transition, withCtx, createCommentVNode, toRefs, onMounted, onUnmounted, watchEffect, Fragment, createVNode, renderList, withKeys, vModelText, mergeModels, useModel, isRef, provide } from 'vue';
 import { i as injectKeyPreviewRef, a as injectKeyProps } from './chunks/types-XqLqBVey.js';
 import { MagicString, babelParse, extractIdentifiers, walkIdentifiers, isStaticProperty, isInDestructureAssignment, walk } from 'vue/compiler-sfc';
