diff --git a/node_modules/vite/dist/node/chunks/dep-3174cbb5.js b/node_modules/vite/dist/node/chunks/dep-3174cbb5.js
index 9027356..bcf08e7 100644
--- a/node_modules/vite/dist/node/chunks/dep-3174cbb5.js
+++ b/node_modules/vite/dist/node/chunks/dep-3174cbb5.js
@@ -31923,12 +31923,14 @@ function buildImportAnalysisPlugin(config) {
                                     addDeps(normalizedFile);
                                 }
                                 const markPos = code.indexOf(preloadMarker, end);
+                                if (markPos > 0) {
                                 s.overwrite(markPos - 1, markPos + preloadMarker.length + 1, 
                                 // the dep list includes the main chunk, so only need to
                                 // preload when there are actual other deps.
                                 deps.size > 1
                                     ? `[${[...deps].map((d) => JSON.stringify(d)).join(',')}]`
                                     : `void 0`);
+                                }
                             }
                         }
                         chunk.code = s.toString();
