diff --git a/dist/node/chunks/dep-557f29e6.js b/dist/node/chunks/dep-557f29e6.js
index 0ef6357f94a83d600c09149205017220e0807ca7..02d665b0e99a98dcc15b2c39692d24e48003203e 100644
--- a/dist/node/chunks/dep-557f29e6.js
+++ b/dist/node/chunks/dep-557f29e6.js
@@ -63427,33 +63427,6 @@ async function bundleConfigFile(fileName, isESM) {
                                 external: true
                             };
                         }
-                        // bundle the rest and make sure that the we can also access
-                        // it's third-party dependencies. externalize if not.
-                        // monorepo/
-                        // ├─ package.json
-                        // ├─ utils.js -----------> bundle (share same node_modules)
-                        // ├─ vite-project/
-                        // │  ├─ vite.config.js --> entry
-                        // │  ├─ package.json
-                        // ├─ foo-project/
-                        // │  ├─ utils.js --------> external (has own node_modules)
-                        // │  ├─ package.json
-                        const idFsPath = path$n.resolve(path$n.dirname(importer), id);
-                        const idPkgPath = lookupFile(idFsPath, [`package.json`], {
-                            pathOnly: true
-                        });
-                        if (idPkgPath) {
-                            const idPkgDir = path$n.dirname(idPkgPath);
-                            // if this file needs to go up one or more directory to reach the vite config,
-                            // that means it has it's own node_modules (e.g. foo-project)
-                            if (path$n.relative(idPkgDir, fileName).startsWith('..')) {
-                                return {
-                                    // normalize actual import after bundled as a single vite config
-                                    path: isESM ? pathToFileURL(idFsPath).href : idFsPath,
-                                    external: true
-                                };
-                            }
-                        }
                     });
                 }
             },